Application/EdFi.Ods.Common/Models/Domain/PropertyType.cs

```diff
@@ -13,20 +13,27 @@ namespace EdFi.Ods.Common.Models.Domain
 {
     public class PropertyType
     {
-        public PropertyType(DbType dbType, int maxLength = 0, int precision = 0, int scale = 0, bool isNullable = false, decimal? minValue = null, decimal? maxValue = null)
+        public PropertyType(DbType dbType, int maxLength = 0, int precision = 0, int scale = 0, bool isNullable = false, decimal? minValue = null, decimal? maxValue = null, int minLength = 0)
         {
             if (maxLength != 0 && (precision != 0 || scale != 0))
             {
-                throw new BadRequestException("Either maxLength or precision/scale can have non-zero values, but not both.");
+                throw new ArgumentOutOfRangeException($"Either maxLength or precision/scale can have non-zero values, but not both.");
```

Comment:
A couple items here:
* As modified, you've added the interpolation symbol (`$`) to the message string, but the string is still a string literal. You should only use interpolation when you're actually including substitutions.
* Replace the inline literal text referencing the parameter names with `{nameof(maxLength)}`, `{nameof(precision)}`, etc.
* When referencing a parameter name in a message like this, you should always use string interpolation and use the `nameof(maxLength)` statement so that the name of parameter is correctly reflected in the message even if it is renamed. Make the change here, and to the new messages added below.

Reply:
I updated it . Thanks